### PR TITLE
2.x: Update Changelog for 2.11.4

### PR DESCRIPTION
**Pull request by enrico-usai**
Add entries from cookbook changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,11 +8,16 @@ CHANGELOG
 - CentOS 8 is no longer supported (EOL on December 31st, 2021).
 - Upgrade Slurm to version 20.11.8.
 - Upgrade Cinc Client to version 17.2.29.
+- Upgrade NICE DCV to version 2021.2-11190.
+- Upgrade NVIDIA driver to version 470.82.01.
+- Upgrade CUDA library to version 11.4.3.
+- Upgrade NVIDIA Fabric manager to `nvidia-fabricmanager-470`.
```
Comment (mauri-melato) [inline CHANGELOG.md]: Align the wording to the the other NVIDIA software components: "...to version x.y.z"

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,11 +8,16 @@ CHANGELOG
...
 - Disable packages update at instance launch time on Amazon Linux 2.
 - Disable unattended packages update on Ubuntu.
+- Install Python 3 version of `aws-cfn-bootstrap` scripts on CentOS 7 and Ubuntu 18.04.
```
Comment (mauri-melato) [inline CHANGELOG.md]: it's worth to add ...aligning with U20 and AL2

Comment (enrico-usai) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (enrico-usai) [reply CHANGELOG.md]: Done

**Review by mauri-melato — APPROVED** [review]

**Pull request merged**